src/main.rs

```diff
@@ -435,17 +447,19 @@ enum Commands {
     /// Creates a producer swarm
     Producers {
         #[clap(short, long)]
-        topic: String,
+        topic_prefix: String,
```

Comment:
Changing `--topic` to `--topic-prefix` arg probalby breaks a bunch of invocations in various places so it's not worth changing. 

Instead I'd either just keep `--topic` and only use `--unique_topics` or just add `--topic-prefix` and then require either `--topic` or `--topic-prefix`.